src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int RANDOM_NUMBER = 101;
+    private ColorSupplier colorSupplier;
+
+    public Lottery() {
+        this.colorSupplier = new ColorSupplier();
```

Comment:
this is redundant here
```suggestion
        colorSupplier = new ColorSupplier();
```